test_land_konsistenz.py

```
@@ -0,0 +1,124 @@
+import asyncio
+from playwright.async_api import async_playwright
+
+async def test_land_konsistenz():
+    print("🌍 LAND-KONSISTENZ TEST")
+    print("======================")
+    
+    async with async_playwright() as p:
+        browser = await p.chromium.launch(headless=False, args=['--no-sandbox'])
+        page = await browser.new_page()
+        
+        console_messages = []
```

Ruff (0.12.2)

Local variable `console_messages` is assigned to but never used

Remove assignment to unused variable `console_messages`

(F841)